lib/govuk_app_config/govuk_error/amqp_user_info_cleanser.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module AMQPUserInfoCleanser
```

Comment:
Oh sorry just noticed another minor thing. This should be under the namespace of GovukError.

No need for the frozen_string_literal comment by the way as we don't have a convention for them in this code base (we [flirted with them in the past](https://github.com/alphagov/content-publisher/pull/1822) to prepare for Ruby 3 but since Ruby 3 didn't have them in the end we dropped them and it's best to avoid them being inconsistently added to apps since it's only a very minor optimisation)